assets/css/aspire-update.css

```diff
@@ -40,22 +40,143 @@
 }
 
 .aspireupdate-notice {
-	background-color: #f0f6fc;
```

Comment:
It looks like `.editorconfig` isn't being read in your IDE. The indent style for CSS is a single tab.